transport_plugins/bled112/scripts/run_mock_observer.py

```diff
@@ -0,0 +1,48 @@
+"""
+Script to test throughput of the gateway
+"""
+
+import argparse
+import time
+import signal
+import sys
+import logging
+import serial
+# TODO import properly, now one have to specify PYTHONPATH=../test
+from util.mock_adv_bled112 import MockAdvertisingBLED112
```

Comment:
Ah, I see the issue now that you were talking about on the call.  I would move `MockAdvertisingBLED112` and its back class in `iotile_transport_bled112/hardware/emulator` and update the unit tests to find the mock there.  That removes the hack needed to import the test code here.

Reply:
Thank you for the suggestion, I was not sure how to resolve it